### ISSUE TITLE
new ticket creation upon user replay

### ISSUE THREAD

**Issue by ranjbarhadi** [ladybirdweb/faveo-helpdesk#1963]
- Faveo Version: Community 1.10.6
- PHP version: 7.1
- Database Driver & Version: MySQL 5.7
- Server specification: Ubuntu 18 with 8 gig ram with SSD

#### Description:
everything works perfect till the user send an email and we answer it. when the user replays to our answer a new ticket will be created which is not the intended behavior. 

I reinstalled my os ( ubuntu 18 ) and reconfigured the server. same unintended behavior happened again.
what should I do? it seems FaveoHelpDesk do not check the subject of emails for the support code.
I also put the support code to be random and not a sequence using this format: $$-### 
ps: our email subject is in farsi

#### Steps To Reproduce:
just do it with current master brach 

#### Downloaded from
- [x] master-branch

- [ ] development-branch

- [ ] release-tag

**Comment by ranjbarhadi** [comment]
solved it. I'll send a pull request any time soon

**Comment by ranjbarhadi** [comment]
I've sent the pull request. hopefully, it would help others :)

**Comment by mverma16** [comment]
@ranjbarhadi which mail client did you face the issue, Gmail?

**Comment by ranjbarhadi** [comment]
@mverma16 yup. it was Gmail. but it doesn't matter now. this patch (#1987) resolve the issue completely
